app/controllers/api/users.php

```diff
@@ -716,6 +715,11 @@
             ->setParam('sessionId', $sessionId)
         ;
 
+        $deletes
+            ->setType(DELETE_TYPE_USERS)
+            ->setDocument($user)
+        ;
+
```

Comment:
Not sure about this addition. I understand it as deleting user when we delete session.